src/Server_TEST.cc

```diff
@@ -1033,6 +1033,79 @@ TEST_P(ServerFixture, AddResourcePaths)
   }
 }
 
+/////////////////////////////////////////////////
+TEST_P(ServerFixture, ResolveResourcePaths)
+{
+  ignition::common::setenv("IGN_GAZEBO_RESOURCE_PATH",
+      "/tmp/some/path:/home/user/another_path");
```

Comment:
just asking, this will only work on Linux, what's about MacOS or WIndows ?

Reply:
Removed the Linux paths in [db49ac7](https://github.com/gazebosim/gz-sim/pull/1508/commits/db49ac7bddd625cec15414dcf51b24eaf31bf3db)